test/unit/handlers/notification/handler.test.js

```diff
@@ -0,0 +1,320 @@
+      transferAmount: {
+        currency: 'USD',
+        amount: '433.88'
+      },
+      ilpPacket: 'AYIBgQAAAAAAAASwNGxldmVsb25lLmRmc3AxLm1lci45T2RTOF81MDdqUUZERmZlakgyOVc4bXFmNEpLMHlGTFGCAUBQU0svMS4wCk5vbmNlOiB1SXlweUYzY3pYSXBFdzVVc05TYWh3CkVuY3J5cHRpb246IG5vbmUKUGF5bWVudC1JZDogMTMyMzZhM2ItOGZhOC00MTYzLTg0NDctNGMzZWQzZGE5OGE3CgpDb250ZW50LUxlbmd0aDogMTM1CkNvbnRlbnQtVHlwZTogYXBwbGljYXRpb24vanNvbgpTZW5kZXItSWRlbnRpZmllcjogOTI4MDYzOTEKCiJ7XCJmZWVcIjowLFwidHJhbnNmZXJDb2RlXCI6XCJpbnZvaWNlXCIsXCJkZWJpdE5hbWVcIjpcImFsaWNlIGNvb3BlclwiLFwiY3JlZGl0TmFtZVwiOlwibWVyIGNoYW50XCIsXCJkZWJpdElkZW50aWZpZXJcIjpcIjkyODA2MzkxXCJ9IgA',
+      condition: 'YlK5TZyhflbXaDRPtR5zhCu8FrbgvrQwwmzuH0iQ0AI'
+    }
+  ],
+  extensionList: {
+    extension: [
+      {
+        key: 'key1',
+        value: 'value1'
+      },
+      {
+        key: 'key2',
+        value: 'value2'
+      }
+    ]
+  }
+}
+
+// Sample Bulk Fulfil Transfer Message send by the Bulk API Adapter Notification Handler as a callback
+const fspiopBulkFulfilTransferMsg = {
+  bulkTransferState: Enum.Transfers.BulkTransferState.COMPLETED,
+  completedTimestamp: new Date().toISOString(),
+  individualTransferResults: [
+    {
+      transferId: fspiopBulkPrepareTransferMsg.individualTransfers[0].transferId,
+      fulfilment: 'tChMwYE5Zw9ZDiU6cUQyeLmReAE55dG74MJKHPWNjvs'
+    }
+  ]
+}
+
+// Sample Kafka protocol message
+const messageProtocol = {
+  id: Uuid(),
+  from: fspiopBulkPrepareTransferMsg.payerFsp,
+  to: fspiopBulkPrepareTransferMsg.payeeFsp,
+  type: 'application/json',
+  content: {
+    headers: {
+      'content-type': 'application/vnd.interoperability.bulkTransfers+json;version=1.1',
+      'fspiop-source': fspiopBulkPrepareTransferMsg.payeeFsp,
+      'fspiop-destination': 'source',
+      date: new Date().toISOString()
+    },
+    payload: fspiopBulkPrepareTransferMsg
+  },
+  metadata: {
+    event: {
+      id: Uuid(),
+      type: Enum.Events.Event.Type.NOTIFICATION,
+      action: Enum.Events.Event.Action.BULK_PREPARE,
+      createdAt: new Date().toISOString(),
+      state: {
+        status: 'success',
+        code: 0
+      }
+    },
+    'protocol.createdAt': new Date().getTime()
+  },
+  pp: ''
+}
+
+// Sample Kafka topic name
+const topicName = 'topic-test'
+
+// Sample Kafka config
+const config = {
+  options: {
+    mode: 2,
+    batchSize: 1,
+    pollFrequency: 10,
+    recursiveTimeout: 100,
+    messageCharset: 'utf8',
+    messageAsJSON: true,
+    sync: true,
+    consumeTimeout: 1000
+  },
+  rdkafkaConf: {
+    'client.id': 'kafka-test',
+    debug: 'all',
+    'group.id': 'central-ledger-kafka',
+    'metadata.broker.list': 'localhost:9092',
+    'enable.auto.commit': false
+  }
+}
+
+Test('Bulk Transfer PREPARE handler', handlerTest => {
+  let sandbox
+  let SpanStub
+  let notificationHandler
+  const postCallbackUrl = 'http://test.local/bulkTransfers'
+  const putCallbackUrl = 'http://test.local/bulkTransfers/{{id}}'
+  // For ErroCallback Scenarios
```

Comment:
```suggestion
  // For ErrorCallback Scenarios
```
nit

Reply:
Sorry, for some reason this did not appear on my screen.

I have created a follow-up PR: https://github.com/mojaloop/bulk-api-adapter/pull/88